File: EJERCICIO_1/EJERCICIOSPHP/salaryBass.php
<?php
class SalaryBass {
    private $valueDay;
    public $workDays;
    private $calculatedValue;
    private $additionalValue;

    public function __construct($valueDay, $workDays) {
        $this->valueDay = $valueDay;
        $this->workDays = $workDays;
    }

    public function getValueDay(){
        return $this->valueDay;
    }
    
    public function getWorkDay() {
        return $this->workDays;
    }

    public function calculateSalary() {
        $this->calculatedValue = $this->valueDay * $this->workDays; 
        if ($this->calculatedValue < 3274050) {
            $this->calculatedValue += $this->additionalValue;
        } 
        return $this->calculatedValue; 
    }
    
}
?>

File: EJERCICIO_1/EJERCICIOJAVA/salaryBass.java
package EJERCICIOJAVA;
public   class salaryBass  {
    private double valueDay;
    public int workDay;
    private double calculate;
    private int additional;

    public salaryBass( double valueDay, int workDay) {
        this.valueDay = valueDay;
        this.workDay = workDay;
    }

    public double getValueDay() {
        return valueDay;
    }
    
    public int getWorkDay() {
        return workDay;
    }
    public double calculateSalary() {
        this.calculate = this.valueDay * this.workDay;
        if (calculate < 3274050) {
            this.calculate = this.calculate + additional;
        } 
        return this.calculate; 
    }
}